frontend/src/lib/utils.tsx

```diff
@@ -1104,7 +1104,7 @@ export function maybeAddCommasToInteger(value: any): any {
     if (!isNumber) {
         return value
     }
-    const internationalNumberFormat = new Intl.NumberFormat('en-US')
+    const internationalNumberFormat = new Intl.NumberFormat('en-US', { maximumFractionDigits: 2 })
```

Comment:
Oh, didn't know we had `maybeAddCommasToInteger()`. I've been using `humanFriendlyNumber()` recently. Perhaps we can unify those.